dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -390,6 +407,96 @@ class BGPPeerMgr(Manager):
         return peers
 
 
+class InterfaceMgr(Manager):
+    def __init__(self, daemon, directory, interface_table = swsscommon.CFG_INTF_TABLE_NAME):
+        super(InterfaceMgr, self).__init__(
+            daemon,
+            directory,
+            [],
+            "CONFIG_DB",
+            interface_table
+        )
+
+    def set_handler(self, key, data):
+        # Interface table can have two keys, 
+        # one with ip prefix and one without ip prefix
+        if '|' in key:
+            data = {}
+            data["interface"], network = key.split('|', 1)
+            try:
+                network = netaddr.IPNetwork(str(network))
+            except:
+                syslog.syslog(syslog.LOG_WARNING, 'Subnet {} format is wrong'.format(network))
```

Comment:
-> Subnet {} format is wrong for interface {}.

Reply:
Done it.